cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -190,15 +190,13 @@ if(builtin_lzma)
     ExternalProject_Add(
       LZMA
       URL ${CMAKE_SOURCE_DIR}/core/lzma/src/xz-${lzma_version}-win32.tar.gz
-      URL_HASH SHA256=c5910475aa8c7b4ed322f3e043c9cc9214e997f2a85b6b32f7702ac6d47364f8
+      URL_HASH SHA256=a923ee68d836de5492d8de0fec467b9536f2543c8579ca11f4b5e6f46a8cda8c
```

Comment:
Is the hash change related to switching the usage from dll to lib? (If not, could be in a separate commit)?

Reply:
This is indeed related (the tar file changed)